src/main/java/seedu/iscam/model/meeting/DateTime.java

```diff
@@ -5,15 +5,19 @@
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
 import java.time.format.DateTimeParseException;
+import java.time.format.ResolverStyle;
 
 /**
  * Represents a meeting's date and time in the iscam book.
  * Guarantees: immutable; is valid as declared in {@link #isStringValidDateTime(String)}
  */
 public class DateTime {
-    public static final String MESSAGE_INVALID_FORMAT = "Date and time should be of the format of dd-MM-yyyy HH:mm.";
+    public static final String MESSAGE_INVALID_FORMAT = "The given date-time is invalid. Possibly due to: \n"
+            + ". Incorrect format (Should be of the format of dd-MM-yyyy HH:mm)\n"
+            + ". Invalid date (e.g. 29-02-2021, a date that is only valid in a leap year)";
```

Comment:
I get the idea but I think this line could be confusing to some people. Could consider removing the year, like "e.g. 29-02, a date only valid in a leap year" or something to that extent?

Reply:
I understand your reasoning but by omitting the year, won't the tester just complain that the message is unclear because `29-02` is not a full date or something.

Reply:
Hmm how about "e.g. 29-02-2021, because 29-02 is not valid in a non-leap year"

Reply:
Sure, I will update to that and merge then